translate_test.go

```diff
@@ -1801,7 +1801,24 @@ func TestRemoveDuplicateFilesAndUnits2_3(t *testing.T) {
 	unitThree.Dropins = append(unitThree.Dropins, dropinThree)
 	testIgn2Config.Systemd.Units = append(testIgn2Config.Systemd.Units, unitThree)
 
-	convertedIgn2Config, err := v23tov30.RemoveDuplicateFilesAndUnits(testIgn2Config)
+	// user test, add a duplicate user and see if it is deduplicated but ssh keys from both are preserved
+	userName := "testUser"
+	userOne := types2_3.PasswdUser{
+		Name: userName,
+		SSHAuthorizedKeys: []types2_3.SSHAuthorizedKey{
+			"one",
+			"two",
+		},
+	}
+	userTwo := types2_3.PasswdUser{
+		Name: userName,
+		SSHAuthorizedKeys: []types2_3.SSHAuthorizedKey{
+			"three",
+		},
+	}
+	testIgn2Config.Passwd.Users = append(testIgn2Config.Passwd.Users, userOne, userTwo)
```

Comment:
It'd be good to add a third user with a different name and make sure it's kept separate in the output.

Reply:
updated with a third user